scripts/deleteAndLoadSnapsets.sh

```diff
@@ -3,7 +3,7 @@ GPII_APP_DIR=${GPII_APP_DIR:-"/app"}
 
 GPII_STATIC_DATA_DIR=${GPII_STATIC_DATA_DIR:-"${GPII_APP_DIR}/testData/dbData"}
 GPII_PREFERENCES_DATA_DIR=${GPII_PREFERENCES_DATA_DIR:-"${GPII_APP_DIR}/testData/preferences"}
-GPII_BUILD_DATA_DIR=${GPII_BUILD_DATA_DIR:-'/tmp/build/dbData'}
+GPII_BUILD_DATA_DIR=${GPII_BUILD_DATA_DIR:-"${GPII_APP_DIR}/build/dbData/snapset"}
```

Comment:
The name of GPII_BUILD_DATA_DIR gives an impression of the build directory at "${GPII_APP_DIR}/build". Might be better to rename it to GPII_SNAPSET_DATA_DIR. Please keep in mind to update the documentation if the renaming does happen. Thanks.

Reply:
Done.  CI passed.